README.md

```diff
@@ -0,0 +1,103 @@
+
+<br />
+
+This is the DEA Notebooks and DEA Tools repository of [Digital Earth Australia](https://www.dea.ga.gov.au/).
+These can be viewed on the [DEA Knowledge Hub](https://docs.dea.ga.gov.au/) and run on the
+[DEA Sandbox](https://app.sandbox.dea.ga.gov.au/) and the [National Computational Infrastructure (NCI)](https://nci.org.au/).
+
+**Contribute** &mdash; DEA Notebooks is open-source and we would love your contribution!
+Learn [how to contribute](#contribute) below.
+
+**License** &mdash; The code in this repository is licensed under the [Apache
+License, Version 2.0](https://www.apache.org/licenses/LICENSE-2.0). DEA
+data is licensed under the [Creative Commons by Attribution 4.0
+license](https://creativecommons.org/licenses/by/4.0/).
+
+**Documentation** &mdash; See the [DEA Notebooks
+Wiki](https://github.com/GeoscienceAustralia/dea-notebooks/wiki).
+
+**Contact** &mdash; For assistance with any of these notebooks and tools,
+please ask a question on our [Open Data Cube Slack
+channel](http://slack.opendatacube.org/) or on the [GIS Stack
+Exchange](https://gis.stackexchange.com/questions/tagged/open-data-cube)
+using the `open-data-cube` tag. You can also [report an issue on this
+repository](https://github.com/GeoscienceAustralia/dea-notebooks/issues).
+
+**Citation** &mdash; If you use this repository in your work, please reference
+it with the following citation.
+
+> Krause, C., Dunn, B., Bishop-Taylor, R., Adams, C., Burton, C., Alger,
+> M., Chua, S., Phillips, C., Newey, V., Kouzoubov, K., Leith, A.,
+> Ayers, D., Hicks, A., DEA Notebooks contributors 2021. Digital Earth
+> Australia notebooks and tools repository. Geoscience Australia,
+> Canberra. <https://doi.org/10.26186/145234>
+
+## Introduction
+
+The Jupyter Notebooks and Python tools in this repository are for
+analysing DEA's satellite data products. The notebooks are designed to demonstrate
+how to use the tools and data to conduct a broad range of geospatial
+analyses. They also demonstrate how to integrate with other open-source
+software such as [Open Data Cube](https://www.opendatacube.org/) and
+[xarray](http://xarray.pydata.org/en/stable/). You can see real-world
+use cases on the [USAGE](https://github.com/GeoscienceAustralia/dea-notebooks/blob/stable/USAGE.rst)
+page.
+
+## Browse the DEA Knowledge Hub
+
+The Knowledge Hub is where you can easily view the notebooks and read DEA Tools documentation. You can browse the following sections.
```

Comment:
Hey @benji-glitsos-ga, thinking about this some more, I think we might want to change these sections up a little. The list of notebook folder dotpoints below applies more specifically to DEA Notebooks than Knowledge Hub itself - I would change the following:

1. Move the notebook folder dotpoints up under "Introduction" (and perhaps rename this folder"), and change their links to relative links that will open directly on the Sandbox and Github
2. Keep the "Browse the DEA Knowledge Hub, but update it to include a short paragraph describing the kind of content it provides - e.g. technical product metadata, DEA user and reference guides, changelog and tech alerts, DEA Tools Python API etc.